### ISSUE TITLE
Deprecate twisted.scripts.tkunzip

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5140]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)| @thijstriemstra reported|
|-|-|
|Trac ID|trac#5140|
|Type|task|
|Created|2011-06-06 19:35:07Z|
|Branch|https://github.com/twisted/twisted/tree/deprecate-tkunzip-5140|

[twisted/scripts/tkunzip.py](twisted/scripts/tkunzip.py) should be deprecated (it was last updated 5 years ago in r15413).

<details><summary>Searchable metadata</summary>

```
trac-id__5140 5140
type__task task
reporter__thijs thijs
priority__low low
milestone__ 
branch__branches_deprecate_tkunzip_5140 branches/deprecate-tkunzip-5140
branch_author__exarkun exarkun
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1307388907000000 1307388907000000
changetime__1318421340000000 1318421340000000
version__None None
owner__exarkun exarkun

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_1"></a>|@exarkun commented|
|-|-|

(In [32814]) Branching to 'deprecate-tkunzip-5140'

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun commented|
|-|-|

(In [32815]) Deprecate it

refs [#5140](https://github.com/twisted/twisted/issues/5140)

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_4"></a>|@thijstriemstra set owner to @exarkun|
|-|-|

Looks good. Perhaps you can add a copyright date and put the docstring on 3 lines in `twisted.scripts.tkunzip` before you merge it.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_5"></a>|@thijstriemstra commented|
|-|-|

Replying to [thijs](#note_4):
> copyright date

~~date~~

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_6"></a>|@exarkun commented|
|-|-|

(In [32821]) test-case-name, copyright statement, properly formatted module docstring

refs [#5140](https://github.com/twisted/twisted/issues/5140)

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_7"></a>|@exarkun set status to `closed`|
|-|-|

(In [32822]) Merge deprecate-tkunzip-5140

Author: exarkun
Reviewer: thijs
Fixes: [#5140](https://github.com/twisted/twisted/issues/5140)

Deprecate `twisted.scripts.tkunzip`.
